ci/terraform/oidc/dynamo-policies.tf

```diff
@@ -1,5 +1,6 @@
 locals {
-  client_registry_kms_key_arn = data.terraform_remote_state.shared.outputs.client_registry_kms_key_arn
+  client_registry_kms_key_arn      = data.terraform_remote_state.shared.outputs.client_registry_kms_key_arn
+  identity_credentials_kms_key_arn = data.terraform_remote_state.shared.outputs.identity_credentials_kms_key_arn
```

Comment:
would be nice to pull these into the shared.tf file with other locals (I've moved in my PR)